apps/console/src/features/users/components/user-profile.tsx

```diff
@@ -994,15 +1004,21 @@ export const UserProfile: FunctionComponent<UserProfilePropsInterface> = (
                     placeholder={ "Select your" + " " + fieldName }
                     type="dropdown"
                     value={ profileInfo.get(schema.name) }
-                    children={ countryList ? countryList.map(list => {
-                        return {
-                            "data-testid": `${ testId }-profile-form-country-dropdown-` +  list.value as string,
-                            flag: list.flag,
-                            key: list.key as string,
-                            text: list.text as string,
-                            value: list.value as string
-                        };
-                    }) : [] }
+                    children={ [ {
+                        "data-testid": `${ testId }-profile-form-country-dropdown-empty` as string,
+                        key: "empty-country" as string,
+                        text: "Select your country" as string,
+                        value: "" as string
+                    } ].concat(
+                        countryList ? countryList.map(list => {
+                            return {
+                                "data-testid": `${ testId }-profile-form-country-dropdown-` +  list.value as string,
+                                flag: list.flag,
+                                key: list.key as string,
+                                text: list.text as string,
+                                value: list.value as string
+                            };
+                        }) : [] ) }
```

Comment:
Shall we format this properly?
```suggestion
                    children={ [ {
                        "data-testid": `${ testId }-profile-form-country-dropdown-empty` as string,
                        key: "empty-country" as string,
                        text: "Select your country" as string,
                        value: "" as string
                    } ].concat(
                        countryList 
                            ? countryList.map(list => {
                                return {
                                    "data-testid": `${ testId }-profile-form-country-dropdown-` +  list.value as string,
                                    flag: list.flag,
                                    key: list.key as string,
                                    text: list.text as string,
                                    value: list.value as string
                                };
                            }) 
                            : [] 
                     ) }
```